examples/wearable/UIComponents/contents/interactive/coverflow/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 	<meta name="viewport" content="width=device-width,user-scalable=no"/>
 	<link rel="stylesheet" href="../../../lib/tau/wearable/theme/default/tau.css">
+	<link rel="stylesheet" href="../../../lib/tau/wearable/theme/default/tau.circle.css">
```

Comment:
I think it's better to add
	<link href="../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
	<link href="../../css/style.css" rel="stylesheet" />
	<link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />